src/common/logger.ts

```diff
@@ -1,6 +1,13 @@
 import * as bunyan from 'bunyan';
 import config = require('./config');
 
+try {
+  // Validate if LOG_LEVEL has valid bunyan logging level
+  config.LOGGING.level = bunyan.resolveLevel(process.env.LOG_LEVEL || config.LOGGING.level);
+} catch(e) {
+  console.error(`Log level "${process.env.LOG_LEVEL}" is not valid logging level. Falling back to "INFO"`);
```

Comment:
That's good, just need to take into account that it won't match the "schema" of the rest of the logs, but I don't think we have a choice here